src/legacy/components/Receive.tsx

```diff
@@ -59,7 +62,9 @@ const AddressBlock = ({
     >
       <AccordionItemHeading>
         <AccordionItemButton className={cstyles.accordionHeader}>
-          {address}
+          <span onClick={() => handleShowPastelPaperWalletModal(address)}>
+            {address}
+          </span>
```

Comment:
Not sure if clicking on the wallet address is a good idea, because this action is already taken: when you click an address, an accordion item expands. Perhaps adding another button next to "Copy address" and "Export Viewing Key" is a good idea?

Reply:
I have added the right click context menu for this one